src/data.js

```diff
@@ -74,3 +74,13 @@ module.exports.Build = Entity.configure({
     return item;
   },
 });
+
+module.exports.OwnersDirectory = Entity.configure({
+  version: 1,
+  partitionKey: Entity.keys.StringKey('owner'),
+  rowKey: Entity.keys.ConstantKey('someConstant'),
+  properties: {
+    installationID: Entity.types.Number,
```

Comment:
Oi wei 😱 You've already told me about that id spelling, and I forgot, it's not good.
(Does it really work perfectly for you though? I rebased the branch, but still getting errors...)